### PR TITLE
chore: bump dependencies

### PR DESCRIPTION
**Pull request by remyleone**
Migration to Terraform sdk v2: https://www.terraform.io/docs/extend/guides/v2-upgrade-guide.html

### CODE REVIEW

**Review by jerome-quere — APPROVED** [review]

File: scaleway/resource_instance_security_group.go [hunk 1]
```diff
@@ -412,15 +412,17 @@ func securityGroupRuleSchema() *schema.Resource {
...
 			},
 			"ip": {
-				Type:         schema.TypeString,
-				Optional:     true,
-				ValidateFunc: validation.SingleIP(),
-				Description:  "Ip address for this rule (e.g: 1.1.1.1). Only one of ip or ip_range should be provided",
+				Type:     schema.TypeString,
+				Optional: true,
+				ValidateFunc: func(i interface{}, s string) ([]string, []error) {
+					return validation.IsIPAddress(i, "server_ips")
```
Comment (kindermoumoute) [inline scaleway/resource_instance_security_group.go:418]: Why do you check for field `server_ips`?

File: scaleway/resource_lb_backend_beta.go [hunk 2]
```diff
@@ -72,8 +72,10 @@ func resourceScalewayLbBackendBeta() *schema.Resource {
 			"server_ips": {
 				Type: schema.TypeList,
 				Elem: &schema.Schema{
-					Type:         schema.TypeString,
-					ValidateFunc: validation.SingleIP(),
+					Type: schema.TypeString,
+					ValidateFunc: func(i interface{}, s string) ([]string, []error) {
+						return validation.IsIPAddress(i, "server_ips")
```
Comment (kindermoumoute) [inline scaleway/resource_lb_backend_beta.go:77]: same question than above

**Review by kindermoumoute — CHANGES_REQUESTED** [review]

File: scaleway/resource_instance_security_group.go (hunk 1 above)
Comment (remyleone) [reply scaleway/resource_instance_security_group.go:418]: Fixed in https://github.com/scaleway/terraform-provider-scaleway/pull/577

File: scaleway/resource_lb_backend_beta.go (hunk 2 above)
Comment (remyleone) [reply scaleway/resource_lb_backend_beta.go:77]: Fixed in https://github.com/scaleway/terraform-provider-scaleway/pull/577

**Pull request merged**